esp/src/src/nls/bs/hpcc.ts

```diff
@@ -520,10 +535,13 @@
     NoWarningFound: "Poruke Upozorenja (warnings) nisu nađene\n",
     NumberofParts: "Broj Dijelova",
     NumberofSlaves: "Broj Izvršilaca",
+    Of: "Od",
+    Off: "Isključeno",
     OK: "OK",
     Oldest: "Najstariji",
     OldPassword: "Stara Lozinka",
     OmitSeparator: "Izostavite Separator",
+    On: "Na",
```

Comment:
not sure what meaning to assume here... I decided the meaning opposite to Off may be appropriate here - pls take it from the update I sent

Reply:
updated.